coopdevs/decidim-module-action_delegator
# Import delegations via CSV

This PR add importation button to import delegations via CSV.

The csv has this format:
`from, to`

### Testing
1. Go to Participants
2. Delegations
3. Select a consultation
4. In delegations list > Import via csv
5. See new delegations in list

### Screenshots
![imagen](https://github.com/coopdevs/decidim-module-action_delegator/assets/75725233/3f5769cc-12d0-4d7f-b632-2f592c86ff86)
